databricks/koalas/series.py

```diff
@@ -317,8 +317,61 @@ def unique(self):
         # Pandas wants a series/array-like object
         return _col(self.to_dataframe().unique())
 
-    @derived_from(pd.Series)
     def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, dropna=True):
+        """
+        Return a Series containing counts of unique values.
+        The resulting object will be in descending order so that the
+        first element is the most frequently-occurring element.
+        Excludes NA values by default.
+        Parameters
+        ----------
+        normalize : boolean, default False
+            If True then the object returned will contain the relative
+            frequencies of the unique values.
+        sort : boolean, default True
+            Sort by values.
+        ascending : boolean, default False
+            Sort in ascending order.
+        bins : Not Yet Supported
+        # TODO: Update Documentation when its supported
+        dropna : boolean, default True
+            Don't include counts of NaN.
+        Returns
+        -------
+        counts : Series
+        See Also
+        --------
+        Series.count: Number of non-NA elements in a Series.
+        Examples
+        --------
+        >>> pdf = ks.DataFrame({'x':[0, 1, np.nan, 0, 2, np.nan, 3, 3]})
```

Comment:
I think the name of the variable will be df instead of pdf for the rest of the examples to work.

`>>> pdf = ks.DataFrame({'x':[0, 1, np.nan, 0, 2, np.nan, 3, 3]})`
`>>> df = ks.DataFrame({'x':[0, 1, np.nan, 0, 2, np.nan, 3, 3]})`

Reply:
Good catch @shril. Thanks for that